src/main.ts

```diff
@@ -114,6 +115,27 @@ async function run() {
 
     const octokit = new GitHub(core.getInput("github_token"));
 
+    if (createAnnotatedTag) {
```

Comment:
I _believe_ inputs are always strings (https://github.com/actions/toolkit/blob/master/packages/core/src/core.ts#L67), so this will be truthy even when passing `"false"`.

```suggestion
    if (createAnnotatedTag === "true") {
```

Reply:
Ohh nice, I didn't know that. I made this change on my fork and tested the false, true and undefined case. All working as expected. Thanks for the info!